src/Command/TranskribusAuthCommand.php

```diff
@@ -88,11 +88,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                         return Command::FAILURE;
                     } else {
                         $accessToken = $content->{'access_token'};
+                        $refreshToken = $content->{'refresh_token'};
                         $io->writeln('');
                         $io->writeln('Your access token is: '.$accessToken);
                         $io->writeln('');
                         $io->writeln('--- Please copy and add it to your .env.local'.
                                                 ' as APP_TRANSKRIBUS_ACCESS_TOKEN ---');
+
+                        $io->writeln('');
+                        $io->writeln('Your refresh token is: '.$refreshToken);
+                        $io->writeln('');
+                        $io->writeln('--- Please copy and add it to your .env.local'.
+                                                ' as APP_TRANSKRIBUS_REFRESH_TOKEN ---');
```

Comment:
It might be good to output exactly what needs to be copied into `.env.local`, so we don't have to copy and paste twice.